pysr/sr.py

```diff
@@ -2586,3 +2583,122 @@ def _mutate_parameter(param_name: str, param_value):
         return False
 
     return param_value
+
+
+class PySRSequenceRegressor(PySRRegressor):
+    def __init__(
+        self,
+        recursive_history_length: int = 0,
+        **kwargs,
+    ):
+        self.recursive_history_length = recursive_history_length
+        super().__init__(**kwargs)
```

Comment:
This line that calls the init of the parent class should go at the top of `__init__`, rather than the bottom.

Reply:
(Just a convention; doesn’t really matter here though)